lib/events/filelog.go

```diff
@@ -60,9 +62,20 @@ func (cfg *FileLogConfig) CheckAndSetDefaults() error {
 	if !utils.IsDir(cfg.Dir) {
 		return trace.BadParameter("path %q does not exist or is not a directory", cfg.Dir)
 	}
+	if cfg.SymlinkDir == "" {
+		cfg.SymlinkDir = cfg.Dir
+	}
+	if !utils.IsDir(cfg.SymlinkDir) {
+		return trace.BadParameter("path %q does not exist or is not a directory", cfg.SymlinkDir)
+	}
 	if cfg.RotationPeriod == 0 {
 		cfg.RotationPeriod = defaults.LogRotationPeriod
 	}
+	// make sure that rotation period is a multiple of one day
+	rotationDays := cfg.RotationPeriod / (24 * time.Hour)
+	if cfg.RotationPeriod != (rotationDays * 24 * time.Hour) {
```

Comment:
If you want to make sure that `cfg.RotationPeriod` is a multiple of "24 hours", you could just do:

```go
if cfg.RotationPeriod % (24 * time.Hour) == 0 { ... }
```

Reply:
great point!